roles/openshift_openstack/tasks/populate-dns.yml

```diff
@@ -72,6 +72,9 @@
     - hostvars[groups.masters[0]].openshift_master_cluster_public_hostname is defined
     - openshift_openstack_num_masters > 1
 
+# TODO(shadower): add the API and apps IP addresses from the Heat stack here.
```

Comment:
why do not we want to implement that in the patch scope?

Reply:
Because we need to read the Heat output in the dynamic inventory.

That's being added in this PR:

https://github.com/openshift/openshift-ansible/pull/6319

If that gets merged first, I'll add it here.

Reply:
Alright, that PR got merged so my next revision will set the DNS records.